examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,80 @@
+  @objc final public let phoneNumber: Swift.String
+  @objc final public let rawId: Swift.String?
+  @objc public init(phoneNumber: Swift.String, rawId: Swift.String? = nil)
+  public static func == (lhs: AzureCommunication.PhoneNumberIdentifier, rhs: AzureCommunication.PhoneNumberIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class MicrosoftTeamsUserIdentifier : ObjectiveC.NSObject, AzureCommunication.CommunicationIdentifier {
+  @objc final public let userId: Swift.String
+  @objc final public let isAnonymous: Swift.Bool
+  @objc final public let rawId: Swift.String?
+  @objc final public let cloudEnviroment: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool = false, rawId: Swift.String? = nil, cloudEnvironment: AzureCommunication.CommunicationCloudEnvironment = .Public)
+  @objc public init(userId: Swift.String, isAnonymous: Swift.Bool)
+  public static func == (lhs: AzureCommunication.MicrosoftTeamsUserIdentifier, rhs: AzureCommunication.MicrosoftTeamsUserIdentifier) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationAccessToken : ObjectiveC.NSObject {
+  @objc final public let token: Swift.String
+  @objc final public let expiresOn: Foundation.Date
+  @objc public init(token: Swift.String, expiresOn: Foundation.Date)
+  @objc deinit
+  @objc override dynamic public init()
+}
+@objc @objcMembers public class CommunicationCloudEnvironment : ObjectiveC.NSObject {
+  @objc public static var Public: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Dod: AzureCommunication.CommunicationCloudEnvironment
+  @objc public static var Gcch: AzureCommunication.CommunicationCloudEnvironment
+  @objc public init(environmentValue: Swift.String)
+  @objc public func getEnvironmentValue() -> Swift.String
+  public static func == (lhs: AzureCommunication.CommunicationCloudEnvironment, rhs: AzureCommunication.CommunicationCloudEnvironment) -> Swift.Bool
+  @objc deinit
+  @objc override dynamic public init()
+}
+public typealias TokenRefresherClosure = (@escaping AzureCommunication.TokenRefreshOnCompletion) -> Swift.Void
+@objc @objcMembers public class CommunicationTokenRefreshOptions : ObjectiveC.NSObject {
+  @objc public init(initialToken: Swift.String? = nil, refreshProactively: Swift.Bool = false, tokenRefresher: @escaping AzureCommunication.TokenRefresherClosure)
+  @objc deinit
+  @objc override dynamic public init()
+}
+public class CommunicationPolicyTokenCredential : AzureCore.TokenCredential {
+  public init(_ credential: AzureCommunication.CommunicationTokenCredential)
+  public func token(forScopes _: [Swift.String], completionHandler: @escaping AzureCore.TokenCompletionHandler)
+  public func validate() throws
+  @objc deinit
+}
```

Comment:
Is this intentionally not available with `@objc`?  Since it is based on AzureCore I would suspect that's the case.

Reply:
Yes we have the Core import wrapped
```swift
#if canImport(AzureCore)
    import AzureCore
#endif
```